### ISSUE TITLE
Usability Test for Msg Tracker MVP

### ISSUE THREAD

**Issue by RakshindaAslam** [CDCgov/prime-reportstream#6953]
## Problem statement
Perform usability testing for Msg ID Search Tool with O&O team to understand if the MVP delivers functionality as expected to allow internal admin to serach messages easily and get the information needed for troubleshooting. 
The purpose of this effort is to identify what updates/new enhancements need to be made to make the tool useful for the  O&O team for reducing the time spent in manual work today.

## What you need to know
This testing effort covers the initiatives for building the internal msg tracker(#6657).
[Mural board with notes from feedback sessions and an overview of findings](https://app.mural.co/t/primedatainput6410/m/primedatainput6410/1671211214584/1ac88bde3c057ca7b43f6a7801025bd7adffb4ee?sender=ua604735c5cf1162b042a4310)

## Acceptance criteria

Performed usability test for the internal Msg Tracking Tool MVP built for O&O admin team to identify:
- Satisfaction level with the MVP as delivered on a scale from 1-5 with both quantitative and qualitative feedback
- Issues/challenges with the UI/functionality of the feature, if any
- Enhancement requests/if any

## To do

- [ ] ...

**Comment by RakshindaAslam** [comment]
@HeatherRienks @fmccaf1 @a-chappelear - for pre-refinement

**Comment by chris-kuryak** [comment]
Removed from epic #6499 as it is already in epic #6992 (which itself is encompassed within epic #6499)

**Comment by fmccaf1** [comment]
I conducted feedback sessions with three O&O engineers: Mo, Brick, and Ott, with Jillian present for note-taking and observing. I added a link to the Mural board where findings are collected to the main ticket text, above. Some additional questions came out of the research that we were unable to answer at this time:

1. What is the complete lifecycle of a ReportStream message? And, does a message's lifecycle always follow the same steps, or are there situations in which a lifecycle may differ from the norm?
2. Can we (are we technically able to) include all steps of the message's lifecycle in the message detail view of this tool? 
3. Will "transport results" field be filled in when the tool is in prod? If not, what needs to happen to fill this field consistently?
4. Lastly, the link to report id was not functioning at the time of these tests. Once the bugs present during testing are addressed, it would be wise to invite O&O engineers back to look at these screens and provide feedback. 

I think we can move forward with designs for now, and save thinking through how to represent the message at different stages of the lifecycle. Question for the team is whether I should move forward with answering these questions next sprint, or if we should save it further down the road. @chris-kuryak
